src/printer.js

```diff
@@ -31,6 +31,13 @@ var namedTypes = types.namedTypes;
 var isString = types.builtInTypes.string;
 var isObject = types.builtInTypes.object;
 
+// Resources for a multi-node ignore state.
+var shouldIgnore = false;
```

Comment:
Will this work when ran against multiple files? What happens if no `prettier-ignore-end` exists?